buildSrc/src/main/groovy/opendcs.publishing-conventions.gradle

```diff
@@ -0,0 +1,31 @@
+plugins {
+    id 'signing'
+    id 'maven-publish'
+}
+
+
+signing {
+    useGpgCmd()    
+}
+
+if (JavaVersion.current() != JavaVersion.VERSION_1_8) {
+    plugins.apply('dev.sigstore.sign')
+    sigstoreSign {
+        oidcClient {
+            gitHub {
+                audience.set("sigstore")
+            }
+
+            web {
+                clientId.set("sigstore")
+                issuer.set("https://oauth2.sigstore.dev/auth")
+            }
+        }
+}
+}
+/*
+tasks.withType(SigstoreSignFilesTask).configureEach {
```

Comment:
should this commented part be a else?

Reply:
No, I should delete it, it was part of the orginal attempt to allow mixed java versioning in the build.